<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>409</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Worker Rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="workers_rate">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Estimated Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="estimated">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>0.00000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Blockchain Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="blockchain_height">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Confirmed Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="confirmed">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>0.00000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Unconfirmed Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="unconfirmed">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>0.00000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Unconfirmed Blocks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="unconfirmed_blocks">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Next Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="next_reward">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>0.00000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Total Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="potential">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.00000000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Block Confirmations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ColorIndicatorLabel" name="confirmations_left">
            <property name="font">
             <font>
              <family>Monospace</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="advanced" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Workers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="statBtn">
           <property name="text">
            <string>Pool Statistics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="graphBtn">
           <property name="text">
            <string>Graph</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tglWidget">
           <property name="text">
            <string>Widget</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="workers">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rate</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shares</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Score</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Loading...</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="fileMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBitMinerStats">
    <property name="title">
     <string>BitMinerStats</string>
    </property>
    <addaction name="actionExportStatistics"/>
    <addaction name="actionRestoreDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionNexusTools"/>
    <addaction name="separator"/>
    <addaction name="actionCredits"/>
    <addaction name="actionDonate"/>
    <addaction name="actionQt"/>
   </widget>
   <widget class="QMenu" name="menuMining">
    <property name="title">
     <string>Mining</string>
    </property>
    <addaction name="actionMinerControl"/>
    <addaction name="actionIdleControl"/>
    <addaction name="actionManageMiners"/>
    <addaction name="separator"/>
    <addaction name="actionNoMinersConfigured"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuCurrency">
     <property name="title">
      <string>Currency</string>
     </property>
     <addaction name="actionBTC"/>
     <addaction name="actionBTC_USD"/>
     <addaction name="actionBTC_EUR"/>
     <addaction name="separator"/>
     <addaction name="actionLTC"/>
     <addaction name="actionLTC_USD"/>
     <addaction name="actionLTC_EUR"/>
    </widget>
    <addaction name="menuCurrency"/>
   </widget>
   <addaction name="menuBitMinerStats"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuMining"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExportStatistics">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Backup Statistics</string>
   </property>
  </action>
  <action name="actionRestoreDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Restore Database</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionNexusTools">
   <property name="text">
    <string>NexusTools</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionManageMiners">
   <property name="text">
    <string>Manage Miners</string>
   </property>
  </action>
  <action name="actionNoMinersConfigured">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Miners Configured</string>
   </property>
  </action>
  <action name="actionMinerControl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Miner</string>
   </property>
  </action>
  <action name="actionIdleControl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Idle Auto-Control</string>
   </property>
  </action>
  <action name="actionBTC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTC</string>
   </property>
  </action>
  <action name="actionBTC_USD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTC-&gt;USD</string>
   </property>
  </action>
  <action name="actionBTC_EUR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BTC-&gt;EUR</string>
   </property>
  </action>
  <action name="actionLTC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LTC</string>
   </property>
  </action>
  <action name="actionLTC_USD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LTC-&gt;USD</string>
   </property>
  </action>
  <action name="actionLTC_EUR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LTC-&gt;EUR</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorIndicatorLabel</class>
   <extends>QLabel</extends>
   <header>src/colorindicatorlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
